portfolio/src/main/webapp/style.css

```diff
@@ -45,6 +25,46 @@ body {
   max-width: 100%;
 }
 
+#navigation {
+  font-size: 5vw;
+  display:grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows:1fr 1fr;
+  color:white;
+}
+
+.navigation-item {
+    display:flex;
```

Comment:
[add a space after `:`](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops), here and elsewhere

Reply:
Resolved in commit ab25cfb

Reply:
Further formatting in commit 7a710ce